index.html

```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
                 <div class="project-image">
                     <img src="./images/project_1.png">
                 </div>
-                <div class="project-title">
+                <div class="project-title pro-title-one">
                     <h2>Tonic</h2>
                 </div>
-                <div class="projectsub-title">
-                    <ul>
+                <div class="projectsub-title pro-sub-one">
+                    <ul class="sub-title-o">
                         <li class="one">Canopy</li>
                         <li>Back end dev</li>
                         <li>2015</li>
                     </ul>
                 </div>
-                <div class="project-description">
+                <div class="project-description pro-desc-one">
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 </div>
-                <div class="project-tag">
-                    <ul>
+                <div class="project-tag pro-tag-one">
+                    <ul class="pro-tag-o">
                         <li>html</li>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
                 </div>
-                <div class="project-button">
+                <div class="project-button pro-btn-one">
                     <button>See Project</button>
                 </div>
+                <div id="project-detail-all">
+                    <div class="project-title">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="projectsub-title">
+                        <ul>
+                            <li class="one">Canopy</li>
+                            <li>Back end dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="project-tag">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
```

Comment:
- [ ] Kindly adjust the spacing between these badges to match the template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215290886-4d55b515-3f81-4c0f-9309-8e873a28c8d1.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215290892-12b36859-a056-40de-864c-012e80800421.png)